invesalius/constants.py

```diff
@@ -823,22 +823,24 @@
 CAM_MODE = True
 
 # Tractography visualization
-N_TRACTS = 100
-PEEL_DEPTH = 5
-MAX_PEEL_DEPTH = 30
-SEED_OFFSET = 15
+N_TRACTS = 200
+PEEL_DEPTH = 10
+MAX_PEEL_DEPTH = 40
+SEED_OFFSET = 30
 SEED_RADIUS = 1.5
 
 # Increased the default sleep parameter from 0.1 to 0.15 to decrease CPU load during navigation.
-SLEEP_NAVIGATION = 0.15
+SLEEP_NAVIGATION = 0.2
 SLEEP_COORDINATES = 0.05
 
-BRAIN_OPACITY = 0.5
+BRAIN_OPACITY = 0.6
 N_CPU = psutil.cpu_count()
-TREKKER_CONFIG = {'seed_max': 1, 'step_size': 0.1, 'min_fod': 0.1, 'probe_quality': 3,
-                  'max_interval': 1, 'min_radius_curv': 0.8, 'probe_length': 0.4,
-                  'write_interval': 50, 'numb_threads': '', 'max_lenth': 200,
-                  'min_lenth': 20, 'max_sampling_step': 100}
+# the max_sampling_step can be set to something different as well. Above 100 is probably not necessary
+TREKKER_CONFIG = {'seed_max': 1, 'step_size': 0.03125, 'min_fod': 0.05, 'probe_quality': 3,
```

Comment:
I think it's better to put one key-value for line, like this:

```python
TREKKER_CONFIG = {
    "seed_max": 1,
    "step_size": 0.03125,
    "min_fod": 0.05,
    "probe_quality": 3,
    "max_interval": 1,
    "min_radius_curvature": 0.625,
    "probe_length": 0.15625,
    "write_interval": 50,
    "numb_threads": "",
    "max_length": 250,
    "min_length": 10,
    "max_sampling_step": 100,
    "data_support_exponent": 0.5,
    "use_best_init": True,
    "init_max_est_trials": 100,
}
```

Reply:
Fixed!